pages/_app.tsx

```diff
@@ -41,8 +42,10 @@ const SaleorApp = ({ Component, pageProps }: AppLayoutProps) => {
   return (
     <AppBridgeProvider>
       <GraphQLProvider>
-        <ThemeProvider overrides={themeOverrides}>
-          {getLayout(<Component {...pageProps} />)}
+        <ThemeProvider overrides={themeOverrides} ssr={true}>
+          <AuthorizedPage>
+            {getLayout(<Component {...pageProps} />)}
+          </AuthorizedPage>
```

Comment:
Please explain - app-template has both Dashboard context (that needs to be checked for iframe etc) but also other pages - like e.g. checkout storefront. So probably we shouldn't check *every* page for this context?

Reply:
Checkout doesn't use template, it's a separate project. app-template is intended to be used only within the Dashboard context as it was decided few weeks ago by Timur and the team. The app-template outside of dashboard is pretty much useless.

Reply:
Ok!